deployability/modules/allocation/README.MD

```diff
@@ -46,30 +46,30 @@ To use this module, you should use a Debian-based system, we recommend using Ubu
 
   These commands must be available to run on the Host Operating System.
 
-#### Use the Allocation module through the Workflow engine
+#### Use the Allocation module through the JobFlow engine
 
 Now, it is possible to use the Worklow engine library to launch the provision module by doing the following steps:
 
-1. Install the Workflow engine library and its launcher:
+1. Install the JobFlow engine and its launcher:
 
   While in wazuh-qa:
 
   ```bash
   cd modules
-  pip3 uninstall -y workflow_engine && pip3 install .
+  pip3 uninstall -y jobflow && pip3 install .
   ```
 
 2. Test Fixture to Execute:
 
       It will be necessary to create a fixture (yaml file) where the infrastructure, provisioning, and tests to be executed will be declared.
 
-      >Note: It is possible to find some fixture examples in [deployability/modules/workflow_engine/examples/](../workflow_engine/examples)
+      >Note: It is possible to find some fixture examples in [deployability/modules/jobflow/examples/](../jobflow/examples)
 
       Example:
 
       ```bash
       version: 0.1
-      description: This workflow is used to test agents' deployment for DDT1 PoC
+      description: This jobflow is used to test agents' deployment for DDT1 PoC
```

Comment:
This YAML

Reply:
Fixed
https://github.com/wazuh/wazuh-qa/blob/943b9b03f61959dbf9a01189bace682b73c5bd1f/deployability/modules/allocation/README.MD?plain=1#L72